azurerm/resource_arm_application_gateway.go

```diff
@@ -25,6 +25,15 @@ func possibleArmApplicationGatewaySslCipherSuiteValues() []string {
 	return cipherSuites
 }
 
+func base64EncodedStateFunc(v interface{}) string {
```

Comment:
is this intentionally here, if not should this go into `deprecated.go`?

Reply:
at the moment it is only used by this resource, so moved it here